homeassistant/components/media_player/plex.py

```diff
@@ -45,24 +43,24 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     def update_devices():
         """ Updates the devices objects. """
         try:
-            devices = plexuser.devices()
+            devices = plexserver.clients()
         except BadRequest:
             _LOGGER.exception("Error listing plex devices")
             return
 
         new_plex_clients = []
         for device in devices:
-            if (all(x not in ['client', 'player'] for x in device.provides)
-                    or 'PlexAPI' == device.product):
+            # For now, let's allow all deviceClass types
+            if device.deviceClass in []:
                 continue
```

Comment:
True, i left that in on purpose.
The original class did some filtering, wanted to leave it unfiltered for tests.

If accepted and confirmed working, i'll remove it